docker-compose.yml

```diff
@@ -40,6 +40,16 @@ services:
 
   alarms:
     image: monasca/alarms:${MON_ALARMS_VERSION}
+    environment:
```

Comment:
This variables should not be here.
When you set `KEYSTONE_DEFAULTS_ENABLED` in Dockerfile to true they are not needed.